features/tracker-allowlist.json

```diff
@@ -1423,6 +1423,17 @@
                     }
                 ]
             },
+            "target-opus.map.fastly.net": {
```

Comment:
@sammacbeth could you validate whether the extension will accept `map.fastly.net` or `fastly.net` here, rather than the hostname

Reply:
Move and confirmed that it works on android.